### PR TITLE
feat: enable whitesource scan

### PR DESCRIPTION
**Pull request by MJSiddu**
Ticket:
https://zenhub.ibm.com/workspaces/symposium-workflow-5c49e21b79c5ae72931da0d6/issues/symposium/track-and-plan/25345

### CODE REVIEW

**Comment by MJSiddu** [comment]
@dacleyra - can you add API_KEY and USER_KEY under Secrets?

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -22,6 +22,24 @@ jobs:
       - name: Build
         run: go build -v ./...
 
+      - name: WhiteSource Unified Agent Scan
+      env:
```
Comment (ab-ibm) [inline .github/workflows/go.yml:26]: Invalid yaml file, looks like env is not properly aligned

**Review by ab-ibm — CHANGES_REQUESTED** [review]

File: .goreleaser.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 before:
   hooks:
-    - go mod tidy
+    - go mod tidy -go=1.16 && go mod tidy -go=1.17
```
Comment (mxpaspa) [inline .goreleaser.yml:3]: @dacleyra I added this to get around a go mod issue

**Review by ab-ibm — APPROVED** [review]

**Comment by MJSiddu** [comment]
LGTM.

**Review by dacleyra — APPROVED** [review]

**Pull request merged**